app/models/ticket.py

```diff
@@ -70,8 +70,12 @@ def has_order_tickets(self):
         Else False.
         """
         from app.helpers.helpers import get_count
-        count = get_count(OrderTicket.query.filter_by(ticket_id=self.id))
-        return bool(count > 0)
+        count =  Order.query.filter( Order.id.in_( OrderTicket.query.with_entities(OrderTicket.order_id).filter_by(ticket_id=self.id).all() ) ).filter_by(status='completed').all()
```

Comment:
Make this statement more clear to understand. Take `OrderTicket.query.with_entities(OrderTicket.order_id).filter_by(ticket_id=self.id).all()` out. 

Use the `get_count` method. We don't want a full select query to run. All we need is the count.